int16 id
---
bool result
